web/content/2.4.md

```diff
@@ -0,0 +1,302 @@
+* Restore katello::qpid_client ([#31693](https://projects.theforeman.org/issues/31693))
+* Increase Artemis disk usage limit ([#31607](https://projects.theforeman.org/issues/31607))
+* Allow changing some Foreman Proxy and Apache hostnames on Katello-based certificates ([#31509](https://projects.theforeman.org/issues/31509))
+* Update candlepin broker.xml for jobs processing ([#31497](https://projects.theforeman.org/issues/31497))
+* Remove digital ocean plugin ([#31490](https://projects.theforeman.org/issues/31490))
+* Drop ssl_protocol parameter on foreman_proxy_content ([#31435](https://projects.theforeman.org/issues/31435))
+* Installer can't install the Ansible plugin on Debian/Ubuntu ([#31430](https://projects.theforeman.org/issues/31430))
+* Disable TLS 1.0 and 1.1 by default in Apache ([#31387](https://projects.theforeman.org/issues/31387))
+* Disable weak ciphers in qpid-router ([#31385](https://projects.theforeman.org/issues/31385))
+* Use Java 11 for Candlepin ([#31346](https://projects.theforeman.org/issues/31346))
+* Foreman Proxy HTTPBoot feature should not require tftp ([#30449](https://projects.theforeman.org/issues/30449))
+* As a developer I'd like to set the exportable paths via installer ([#30436](https://projects.theforeman.org/issues/30436))
+* Provide clearer Smart Proxy registration errors with as much context as possible based on the error ([#30284](https://projects.theforeman.org/issues/30284))
+
+#### Installer - foreman-installer script
+* Ensure postgresql-docs and postgresql-contrib are removed as part of postgresql upgrade ([#31671](https://projects.theforeman.org/issues/31671))
+* Migrate ssh_args to ansible default for current installations ([#31553](https://projects.theforeman.org/issues/31553))
+* Incorrect legacy dir in Pulpcore migration ([#31548](https://projects.theforeman.org/issues/31548))
+
+#### Packaging
+* smart_proxy_container_gateway needs to be added to foreman-packaging ([#31643](https://projects.theforeman.org/issues/31643))
+* update of DNS records fails with IPv6 address - invalid rdata format: bad dotted quad ([#31598](https://projects.theforeman.org/issues/31598))
+* rubygem-rkerberos: libkadm5clnt_mit.so will be a problem again in RHEL-8.3 ([#30653](https://projects.theforeman.org/issues/30653))
+
+#### Packaging - Debian/Ubuntu
+* Drop sysvinit scripts ([#31393](https://projects.theforeman.org/issues/31393))
+
+#### Smart Proxy - BMC
+* Expose installed BMC providers as capabilities or settings ([#31537](https://projects.theforeman.org/issues/31537))
+
+#### Smart Proxy - Plugins
+* Drop deprecated disable_plugin methods ([#31303](https://projects.theforeman.org/issues/31303))
+
+*A full list of changes in 2.4.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12][]=1319)*
+
+
+### Katello headline features
```

Comment:
This section seems like it should be toward the top of the list of redmine issues since it's the highlights. Not sure if you were following some convention here though

Reply:
Did you mean the "Katello headline features"? I was putting the Katello items below the Foreman section as they were separate before (3.18 and prior). The general formatting of the Katello section was modeled on the current 3.18 release notes.